src/util/processing.cpp

```diff
@@ -24,9 +26,7 @@ void process_projection(bulk::world& world, int rows, int cols, float* data,
     int first_row = s * block_size;
     int final_row = std::min((s + 1) * block_size, rows);
 
-    int count = proj_id_max - proj_id_min + 1;
-
-    for (int proj = 0; proj < count; ++proj) {
+    for (int proj = proj_id_min; proj < proj_id_max + 1; ++proj) {
```

Comment:
I don't think I understand this change. The `data` buffer has size `count =  proj_id_max - proj_id_min + 1`. This is what was called the `group_size`; the number of projections that are buffered on the CPU. Did you change the buffer to be a full projection stack on the CPU as well?

Reply:
No, buffer on the CPU is of size `update_every` in continuous mode. But if `group_size` is smaller, then only part of the CPU buffer needs to be processed. Hence the loop should be over the group in the CPU buffer and not over [0, `count`[, where count equals the group size.

Reply:
Basically this does nothing more than that it allows the starting index in the data buffer to be different. Its necessary if you do not want to process the full `data`.